components/admin/lessons/AdminModuleInputs.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useState } from 'react'
+import {
+  AddModuleMutation,
+  UpdateModuleMutation,
+  useAddModuleMutation,
+  useUpdateModuleMutation
+} from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, MD_INPUT, TextField } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminModuleInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { get } from 'lodash'
+import { RefetchQueriesFunction, ApolloError } from '@apollo/client'
+
+type Module = { id: number; name: string; content: string }
+
+type Props = {
+  lessonId: number
+  refetch: RefetchQueriesFunction
+  title?: string
+  onAddModule?: (
+    m:
+      | (AddModuleMutation['addModule'] & { lesson: { id: number } })
```

Comment:
I had to define the lesson ID field because `UpdateModuleMutation` and [`AddModuleMutation`](https://github.com/flacial/c0d3-app/blob/e0515ccd9df5f9ece6b52eed8e19b55f691a09ac/graphql/index.tsx#L458) doesn't have it.